lib/registries/addon/overview/-components/diff-manager/component.ts

```diff
@@ -44,24 +44,16 @@ export default class DiffManager extends Component<Args> {
     @task
     @waitFor
     async loadRevision(registration: RegistrationModel, headRevisionId?: string, baseRevisionId?: string) {
-        const revisions = await registration.queryHasMany('schemaResponses', {
-            filter: {
-                reviews_state: [
-                    RevisionReviewStates.Approved,
-                    RevisionReviewStates.Unapproved,
-                    RevisionReviewStates.RevisionPendingModeration,
-                ],
-            },
-        });
         if (baseRevisionId) {
             let baseRevision = this.store.peekRecord('schema-response', baseRevisionId);
             if (!baseRevision) {
                 baseRevision = await this.store.findRecord('schema-response', baseRevisionId);
             }
             this.baseRevision = baseRevision;
         } else {
-            this.baseRevision = revisions.lastObject;
+            this.baseRevision = await registration.originalResponse;
         }
+        const revisions = await registration.queryHasMany('schemaResponses');
```

Comment:
Do we need to load this anymore?

Reply:
Yes we do. We still can't rely on `latest_response` since it will only show the latest approved.